Update documentation for HomeKit Bridge (home-assistant#26791)

mback2k opened the pull request:
**Description:**
- Add documentation for advertise_ip option
- Add documentation section for Docker Network Isolation
- Update homekit.markdown

**Pull request in home-assistant:** home-assistant/home-assistant#26791

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

mback2k committed:
Update documentation for HomeKit Bridge (home-assistant#26791)

- Add documentation for advertise_ip and advertise_mac options
- Add documentation section for Docker Network Isolation
- Update homekit.markdown

frenck committed:
:pencil2: Tweaks

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.

mback2k committed:
Remove the possibility to set the MAC address of the HomeKit Bridge

Since the MAC address is a random device ID, there is no need
for the user to be able to set a custom MAC address value for it.

This change was requested in the parent PR home-assistant#26791.

mback2k committed:
:pencil2: Tweaks

mback2k commented:
I had to make some changes to the documentation due to https://github.com/home-assistant/home-assistant/pull/26791#issuecomment-543899797.

frenck reviewed (dismissed):
LGTM!

Thanks, @mback2k! 👍
